website/docs/sql/command/copy_to.md

```diff
@@ -29,6 +29,13 @@ Depending on the target location, Hyper will write data, e.g., to a
 local file, an AWS S3 bucket or standard output. More information
 on the available locations can be found in [External Locations](/docs/sql/external/location).
 
+Since `COPY TO` can overwrite existing files on disk or an S3 bucket,
+it is easy to accidentally remove any existing data, so you will need to
+have superuser privileges. Without superuser privileges, it is also
```

Comment:
We should not mention "super user" privileges here. The user *does not* need super user privileges as in "he does not need to be `root` on his Linux session" and we never introduced the concept of user management to the Hyper API users anywhere...